Sources/MongoSwift/BSON/BSONEncoder.swift

```diff
@@ -149,7 +149,7 @@ public class BSONEncoder {
 
         let encoder = _BSONEncoder(options: self.options)
 
-        guard let boxedValue = try encoder.box_(value) else {
+        guard let boxedValue = try? encoder.box_(value) else {
```

Comment:
I don't think we should swallow the actual thrown error here, we lose information about what went wrong. I think we should try/catch and if we catch something you can put the resulting error info into the `debugDescription`.